R/use_extendr.R

```diff
@@ -175,6 +184,14 @@ use_extendr <- function(path = ".",
     overwrite = overwrite
   )
 
+  # add config.R template
+  use_rextendr_template(
+    "config.R",
+    save_as = file.path("tools", "config.R"),
+    quiet = quiet,
+    overwrite = overwrite
+  )
```

Comment:
the `config.R` is used instead of a `configure.win` and `configure` file. Making it easier to maintain (plus its R which is just nicer to use imo).